broker/src/main/java/io/zeebe/broker/Broker.java

```diff
@@ -117,6 +118,13 @@ public void addPartitionListener(final PartitionListener listener) {
     return startFuture;
   }
 
+  public boolean isHealthy() {
```

Comment:
Can we also add the TODO comment pointing to the follow up issue where we can remove this (since we will use the HTTP endpoint?).

Reply:
Or would it make sense for the `Broker` to also be `HealthMonitorable`? :thinking: